jupyterlab/pyproject.toml

```diff
@@ -60,5 +60,11 @@ npm = ["jlpm"]
 source_dir = "src"
 build_dir = "qsharp-jupyterlab/labextension"
 
+[tool.hatch.build.targets.wheel]
+packages = ["qsharp-jupyterlab"]
```

Comment:
This directory does not exist, are you sure this is correct?

Reply:
that folder is a child folder under jupyterlab, and from testing locally it looks like it pulls in the right contents.

Reply:
Sorry to bother here, I'm just trying to help! Which of the following mappings do you want the wheel to use as a top-level directory?

1. `jupyterlab/qsharp-jupyterlab` ->`qsharp-jupyterlab` (`packages = ["jupyterlab/qsharp-jupyterlab"]`)
2. `jupyterlab` ->`jupyterlab` (`packages = ["jupyterlab"]`)

Reply:
when I unzip the wheel, the contents look like:
![image](https://github.com/microsoft/qsharp/assets/10567287/db54d9d1-2fd6-4049-806e-609f2c461988)
this matches the contents of the wheel that we have before the upgrade to hatch 1.19, so I think this is intentional.

Reply:
Okay thank you! I would strongly recommend the following config if it produces the contents that you expect:

```toml
[tool.hatch.build.targets.wheel]
packages = ["jupyterlab/qsharp-jupyterlab"]
```

Reply:
Oh I see what's happening, there are multiple project files in this repository and I was not looking at where it was located. My apologies! Everything looks great